server/service/src/sync/translations/name.rs

```diff
@@ -149,7 +149,7 @@ impl SyncTranslation for NameTranslation {
 
     fn try_translate_from_upsert_sync_record(
         &self,
-        connection: &StorageConnection,
+        _: &StorageConnection,
```

Comment:
I'm curious is there a need to keep these fields in when they aren't used?

Reply:
They are defined in the SyncTranslation trait. You have to have this parameter in the impl to fulfil the "trait contract" (with the incorrect fn signature your struct doesn't impl the trait).